templates/swift/Sources/Enums/Enum.swift.twig

```diff
@@ -0,0 +1,5 @@
+enum {{ enum.name | caseUcfirst | overrideIdentifier }} : String {
+{% for value in enum.enum %}
+  {% set key = enum.keys is empty ? value : enum.keys[loop.index0] %}
+  case {{ key | caseCamel | replace({'-': ''})}} = '{{ value }}'
```

Comment:
Since we defined the raw type as `String`, we can remove the ` = '{{ value }}'` as the value can be inferred by the name

Reply:
It's necessary cause the `key`  and `value` have different casing. key has `camelCase` while the value could be lowercase, snakeCase , kebabCase etc

Reply:
Ah yes true :+1: